src/components/select/select.component.spec.ts

```diff
@@ -361,6 +361,34 @@ describe('Select Component - NgModel Input', () => {
 
         expect(component.onValueChange).not.toHaveBeenCalled();
     });
+
+    it('should change from untouched to touched when clicking the input in single mode', async () => {
+        component.multiple = false;
+        component.value = component.options[0];
+        fixture.autoDetectChanges();
```

Comment:
`autoDetectChanges` is designed to configure how change detection should be run rather than actually running change detection (it does trigger change detection which is why it is working, however this is not what is should be used for, we have `fixture.detectChanges` for that instead). If your tests need this enabled then we should add this to the `beforeEach`, eg:

![image](https://user-images.githubusercontent.com/20795331/70802361-2f644380-1da9-11ea-8078-094d1bccca61.png)

Then in our tests we should be able to use:

`fixture.detectChanges()` as this is the function that actually does the change detection., eg:

![image](https://user-images.githubusercontent.com/20795331/70802406-502c9900-1da9-11ea-9186-64054d1420eb.png)